source/get-started/download-and-install.txt

```diff
@@ -16,20 +16,21 @@ Download and Install
 
    .. step:: Install dependencies
 
-      Before you being developing, ensure you have the following dependencies
+      Before you being this tutorial, ensure you have the following dependencies
       installed in your development environment:
       
       - Compiler that supports C++17, such as `GCC <https://gcc.gnu.org/install/>`__, `Clang <https://clang.llvm.org/>`__,
         or `Visual Studio <https://visualstudio.microsoft.com/>`__
       - `CMake <https://cmake.org/>`__ v3.15 or later
       - `pkg-config <https://www.freedesktop.org/wiki/Software/pkg-config/>`__
 
-      .. note:: C++17 Polyfills
+      .. note:: Pre-C++17 Configurations
 
          This tutorial configures the {+driver-short+} to use the C++17 standard
-         library. If you want to install the driver for pre-C++17 configurations,
-         you must choose a C++17 polyfill library. To learn more about C++17
-         polyfills, see the :ref:`cpp-polyfill-selection` guide.
+         library, but C++11 is the minimum supported language version. If you want
+         to install the driver for pre-C++17 configurations, set the ``CMAKE_CXX_STANDARD``
```

Comment:
Wording tweak suggestion to include a reference to the "Important" note regarding using C++17 or newer:

> Although C++11 is the minimum supported language version, this tutorial configures the {+driver-short+} with the C++17 standard, as recommended by <link to "C++17 Polyfill Configuration">. If you want to configure the driver for a pre-C++17 standard, set the ``CMAKE_CXX_STANDARD`` configuration option accordingly. The driver will automatically use bsoncxx library polyfill implementations for required C++17 features.